src/components/Table/TableBody/TableBodyCell/TableBodyCell.test.tsx

```diff
@@ -14,7 +14,7 @@ describe('TableCell', () => {
       expect(tableCell).toBeInTheDocument();
     });
 
-    test('td element is rendered with specific width style atrtibute based on width prop', () => {
```

Comment:
This extension is priceless: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker